whisper.cpp

```diff
@@ -3854,7 +3854,7 @@ int whisper_full(
                         return a.sequence.sum_logprobs_all > b.sequence.sum_logprobs_all;
                     });
 
-                    int cur_c = 0;
+                    signed int cur_c = 0;
```

Comment:
Hmm, I think you meant:

```suggestion
                    unsigned int cur_c = 0;
```